### PR TITLE
Williamhe/1699247 add mongodb to php fpm images

### PR DESCRIPTION
**Pull request by william-msft**
- [x] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1699247
  - Adding mongodb module to php-fpm
- [x] Tests are included and/or updated for code changes.
- [x] Proper license headers are included in each file.

### CODE REVIEW

**Review by daniv-msft — DISMISSED** [review]
Looks good to me

File: images/runtime/php-fpm/template.base.Dockerfile [hunk 1]
```diff
@@ -78,6 +78,13 @@ RUN pecl install redis && docker-php-ext-enable redis
 # https://github.com/Imagick/imagick/issues/331
 RUN pecl install imagick && docker-php-ext-enable imagick
 
+# deprecated from 5.*, so should be avoided 	
+RUN set -eux; \	
+    if [[ $PHP_VERSION != 5.* && $PHP_VERSION != 7.0.* ]]; then \	
+        echo "pecl/mongodb requires PHP (version >= 7.1.0, version <= 7.99.99)"; \
```
Comment (pauld-msft) [inline images/runtime/php-fpm/template.base.Dockerfile:84]: very minor nitpick: this comment/echo statement is no longer true, as we can have mongodb >= 8. Can we remove this or update it to be more accurate?

**Review by pauld-msft — DISMISSED** [review]
:shipit:

File: images/runtime/php-fpm/template.base.Dockerfile (hunk 1 above)
Comment (william-msft) [reply images/runtime/php-fpm/template.base.Dockerfile]: `echo` removed. Thanks!

**Review by snehapar9 — APPROVED** [review]

**Pull request merged**